### DISCUSSION TITLE
v5.2.6

### DISCUSSION THREAD

**Discussion by Mygod** [shadowsocks/shadowsocks-android#2804] [General]
* Support Android 12. (#2749)
* Fixes. (#2786, #2791, #2803)

<hr /><em>This discussion was created from the release <a href='https://github.com/shadowsocks/shadowsocks-android/releases/tag/v5.2.6'>v5.2.6</a>.</em>

**Comment by mmofaha** [comment]
Sir please...
I would like to request for button/menu as per screenshot attached.

TQVM S
![photo_2021-09-19_18-08-24](https://user-images.githubusercontent.com/88963369/133923647-ee7fc5d7-ee6f-4707-b019-c99aa9f2c93f.jpg)

**Comment by javadz501** [comment]
Please update the Shadowsocks FOSS app as well, even 32-bit and 64-bit files are not available for it, just a very old universal file!
It has a much more beautiful design :)
